sloglint.go

```diff
@@ -44,6 +47,13 @@ func New(opts *Options) *analysis.Analyzer {
 	}
 }
 
+const (
+	snakeCase  = "snake"
+	kebabCase  = "kebab"
```

Comment:
Is it worth supporting the upper case variants of these too by accepting SNAKE or KEBAB as valid values? I don't personally need them, but they seem to be supported by the library chosen for this so it's virtually free.

Reply:
Good point! To be honest, I've never seen anything other than snake and kebab cases in logs 😅 Let's support just these for now, like you said, it's super easy to add more if somebody needs them.